_posts/2017-11-13-Dragons-breathe.markdown

```diff
@@ -0,0 +1,20 @@
+---
+layout: post
+title:  "Dragon’s breath"
```

Comment:
Dragon's Breath is capitalized

Reply:
I know it's lowercase on the Sorcerer and Wizard lists, but in the spell description it's capitalized.